src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -297,8 +298,8 @@ public final void visitToken(DetailAST blockCommentNode) {
             // store as field, to share with child Checks
             context.get().blockCommentAst = blockCommentNode;
 
-            final String treeCacheKey = blockCommentNode.getLineNo() + ":"
-                    + blockCommentNode.getColumnNo();
+            final LineColumn treeCacheKey = new LineColumn(blockCommentNode.getLineNo(),
+                    blockCommentNode.getColumnNo());
 
             final ParseStatus result;
```

Comment:
This code has `containsKey` and `get` but is not reported by MUI_CONTAINSKEY_BEFORE_GET.
https://github.com/checkstyle/checkstyle/pull/11146